src/main/java/Duke.java

```diff
@@ -1,10 +1,165 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import java.util.Scanner;
+
+
+public class Duke {
+    public static void printHorizontal () {
+        System.out.println("____________________________________________________________");
+    }
+    public static void printHeader() {
+        printHorizontal();
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        printHorizontal();
+    }
+    public static String extractKeyword (String command) {
+        String[] parsedStrings = command.split(" ", 2);
+        return parsedStrings[0];
+    }
+    public static String extractDetails (String command) {
+        String[] parsedStrings = command.split(" ", 2);
+        return parsedStrings[1];
+    }

```

Comment:
I agree with @yujinyang1998 - might want to consider renaming these two functions, as it is not clear what exactly the functions are performing.